added metadata section (#2077)
SumoLogic/sumologic-documentation: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/citrix-cloud-source.md

The Citrix Cloud API integration retrieves system logs every 5 minutes.

## Metadata Fields

> Metadata fields will be set, if the integration is configured with the SIEM forward option. See **Metadata Fields** table below:

Reviewer: ```suggestion
The following **Metadata Fields** will be set if you configure your integration with the SIEM forward option.
```

| Field Name | Value |

> | --- | --- |

Reviewer 2: ```suggestion
| :--- | :--- |
```

Reviewer 2: Columns should be left-aligned as per the style guide.
